### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Diliaver**

### CODE REVIEW

File: db/models.py [hunk 1]
```diff
@@ -45,3 +50,56 @@ class MovieSession(models.Model):
...
+                         f'be in available range: '
+                         f'(1, seats_in_row): (1'
+                         f', {self.movie_session.cinema_hall.seats_in_row})']})
+
+        if not self.row <= self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                'row': [f'row number must be '
+                        f'in available range: (1, rows): (1'
+                        f', {self.movie_session.cinema_hall.rows})']})
+
```
Comment (Abnormaltype) [inline db/models.py:95]: use "" instead of ''

File: services/order.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                                  row=ticket["row"],
+                                  seat=ticket["seat"])
+
+        return order
+
+
+def get_orders(username=None):
+    if username:
+        return Order.objects.filter(user__username=username)
+    else:
```
Comment (Abnormaltype) [inline services/order.py:25]: remove unnecessary else

File: services/user.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+def update_user(user_id, username=None, password=None,
+                email=None, first_name=None, last_name=None):
+    if username:
+        User.objects.filter(id=user_id).update(username=username)
+    if email:
+        User.objects.filter(id=user_id).update(email=email)
+    if password:
+        u = User.objects.get(id=user_id)
```
Comment (Abnormaltype) [inline services/user.py:26]: u - not informative name for a variable

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: db/models.py [hunk 4]
```diff
@@ -45,3 +50,56 @@ class MovieSession(models.Model):
...
+                'seat': [f"seat number must "
+                         f"be in available range: "
+                         f"(1, seats_in_row): (1"
+                         f", {self.movie_session.cinema_hall.seats_in_row})"]})
+
+        if not self.row <= self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                'row': [f"row number must be "
+                        f"in available range: (1, rows): (1"
+                        f", {self.movie_session.cinema_hall.rows})"]})
```
Comment (Abnormaltype) [inline db/models.py:94]: you didn't fix that

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: db/models.py (hunk 4 above)
Comment (Diliaver) [reply db/models.py]: Sorry(

**Review by Abnormaltype — APPROVED** [review]
Well done!

**Pull request closed without merging**